deployment/ansible/group_vars/all.example

```diff
@@ -49,7 +49,7 @@ redis_bind_address: "redis.service.driver.internal"
 redis_port: 6379
 
 ## WEB SETTINGS
-js_html5mode: "true"
+js_html5mode: "false"
```

Comment:
There was some discussion about this in the DRIVER Slack channel -- it seems like Tiago ran into a similar issue. I'm still not sure why it was a problem, but the app was failing to load with HTML5 enabled due to this issue: https://stackoverflow.com/questions/37034140/location-in-html5-mode-requires-a-base

Reply:
Yeah, good call; this has tripped up several people recently; I don't know why it broke in the first place but our standard practice has become setting it to false, so it should be that way here too.